makefiles/pexec.mk

```
@@ -0,0 +1,43 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += pexec
+PEXEC_VERSION := 1.0rc8
+DEB_PEXEC_V   ?= $(shell $(SED) 's/rc/~rc/g' <<< $(PEXEC_VERSION))
+
+pexec-setup: setup
+	wget -q -nc -P$(BUILD_SOURCE) http://ftp.gnu.org/gnu/pexec/pexec-$(PEXEC_VERSION).tar.gz
```

I can't.. I just can't... (Download the signature of the tarball)

```suggestion
	wget -q -nc -P $(BUILD_SOURCE) http://ftp.gnu.org/gnu/pexec/pexec-$(PEXEC_VERSION).tar.gz{,.sig}
```

---

Also, use `ftpmirror.gnu.org` not `ftp.gnu.org`.

---

I updated my suggested change; should reflect that change if the suggested change is accepted.